go.mod

```diff
@@ -358,7 +369,7 @@ replace (
 	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.25.1
 	k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.25.1
 	k8s.io/sample-controller => k8s.io/sample-controller v0.25.1
+	kubevirt.io/containerized-data-importer-api v1.57.0-alpha1 => kubevirt.io/containerized-data-importer-api v1.56.1
```

Comment:
Any reason why we did this?

Reply:
This is required for dependency mapping with k8s.io/api version we currently support.